global.json

```diff
@@ -30,6 +31,23 @@
 					"title": "Allwed vxlan range",
 					"format": "^([0-9]{1,8}?-[0-9]{1,8}?)$"
 				}
+			},
+			"operProperties": {
+				"numNetworks": {
+					"type": "int"
+				},
+				"defaultNetwork": {
```

Comment:
I'm not sure if you want this at global level. It probably needs to be per tenant